filter_plugins/oo_filters.py

```diff
@@ -71,6 +72,44 @@ def oo_merge_dicts(first_dict, second_dict):
         return merged
 
     @staticmethod
+    def oo_merge_hostvars(hostvars, variables, inventory_hostname, ansible_version):
```

Comment:
Can ansible_version be accessed through the ansible api rather than being passed in?